api/types/installers/agentless-installer.sh.tmpl

```diff
@@ -12,6 +12,11 @@ set -o nounset
 
   . /etc/os-release
 
+  PACKAGE_LIST="{{ .TeleportPackage }}"
+  if [[ "{{ .AutomaticUpgrades }}" == "true" ]]; then
+    PACKAGE_LIST="${PACKAGE_LIST} {{ .TeleportPackage }}-updater"
```

Comment:
Is there any condition where `{{ .TeleportPackage }}` is set to anything but `teleport-ent-updater`? We don't currently publish separate packages for OSS and this will error if it's set to `teleport-updater`.

Reply:
It won't :+1: 

`AutomaticUpgrades` comes from:
```
automaticUpgrades := feats.AutomaticUpgrades && modules.GetModules().BuildType() == modules.BuildEnterprise
```

So, it will only be true for Enterprise builds
I left a comment so that we can comeback to this and simplify the condition only:
```
// TODO(marco): remove BuildType check when teleport-upgrade (oss) package is available in apt/yum repos
```